relay-server/src/endpoints/security_report.rs

```diff
@@ -25,19 +25,23 @@ struct SecurityReportParams {
     meta: RequestMeta,
     #[from_request(via(Query))]
     query: SecurityReportQuery,
-    body: BytesWrapper,
+    body: InstrumentedBytes,
```

Comment:
```suggestion
    #[from_request(via(InstrumentedBytes))]
    body: Bytes,
```

Reply:
nice! done in [fff712c](https://github.com/getsentry/relay/pull/2157/commits/fff712ccf76b76f3fcda4b7db7f67c0e6289ca1e)